feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/ui/QuickSettingsComponents.kt

```diff
@@ -54,13 +55,13 @@ import kotlin.math.min
 //TODO: Implement Set Ratio
 @Composable
 fun ExpandedQuickSetRatio(
```

Comment:
we'll also want to indicate which ratio is currently selected when the options are expanded.

for QuickSetRatio Composable function, you can add a `isHighlighted: boolean = false` parameter, and set the `QuickSettingUiItem` subcomponent's isHighlighted value to that. 

when creating the expanded Ratio buttons, we can set the highlight condition as currentRatio == settingRatio
```
@Composable
fun ExpandedQuickSetRatio(
    setRatio: (aspectRatio: Int) -> Unit,
    currentRatio: Int
) {
    val buttons: Array<@Composable () -> Unit> = arrayOf(
        { QuickSetRatio(onClick = { setRatio(1) }, ratio = 1, isHighLighted = currentRatio == 1) },
        { QuickSetRatio(onClick = { setRatio(2) }, ratio = 2, isHighLighted = currentRatio == 2) },
        { QuickSetRatio(onClick = { setRatio(3) }, ratio = 3, isHighLighted = currentRatio == 3) }

    )
    ExpandedQuickSetting(quickSettingButtons = buttons)
}

//TODO: Implement Set Ratio
@Composable
fun QuickSetRatio(onClick: () -> Unit, ratio: Int, isHighLighted: Boolean = false) {
    val enum = when (ratio) {
        1 -> CameraAspectRatio.THREE_FOUR
        2 -> CameraAspectRatio.NINE_SIXTEEN
        3 -> CameraAspectRatio.ONE_ONE
        else -> CameraAspectRatio.ONE_ONE
    }
    QuickSettingUiItem(
        drawableResId = enum.getDrawableResId(),
        text = stringResource(id = enum.getTextResId()),
        accessibilityText = stringResource(id = enum.getDescriptionResId()),
        onClick = { onClick() },
        isHighLighted = isHighLighted
    )
}
```